### PR TITLE
feat: changes to move Traces up front and clarify docs

### PR DESCRIPTION
**Pull request by MariadeAnton**
* moves Traces from integrations to core platform
* creates 3 paths for people to follow, new to OTel, export and import

## Affected Components
* [X] Docs

## Pre-Requisites
* [x] Code is linted (`npm run lint`)

### Previous
image 

### Proposed
image

### CODE REVIEW

File: site/content/docs/traces-open-telemetry/_index.md [hunk 1]
```diff
@@ -1,47 +1,55 @@
...
 beta: true
 ---
 
 {{< info >}}
-The OpenTelemetry integration is currently in **beta** and available for free on all plans. 
-Need help getting started? We offer a **free, 1:1 valet onboarding service** to help you instrument your stack with the 
+Checkly traces are OpenTelemetry native. Currently in **beta** and available for free on all plans. 
+Need help getting started? We offer a **free, 1:1 onboarding service** to help you instrument your stack with the 
 correct OpenTelemetry SDKs and enable tracing inside Checkly. 
 [Book an onboarding session right here](https://calendly.com/maria-checkly/traces-onboarding)
```
Comment (sujaya-sys) [inline site/content/docs/traces-open-telemetry/_index.md:15]:
```suggestion
[Book an onboarding session right here](https://calendly.com/maria-checkly/traces-onboarding).
```

File: site/content/docs/traces-open-telemetry/importing-traces/_index.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+menu:
+  platform:
+    parent: "Traces (beta)"
+beta: true
+---
+
+Send your 3rd party backend OpenTelemetry traces to Checkly. Inspect and drill down to why a check failed or degraded.
+<!--more-->
+
+In [the Traces tab your Checkly account settings](https://app.checklyhq.com/settings/account/traces):
```
Comment (drakirnosslin) [inline site/content/docs/traces-open-telemetry/importing-traces/_index.md:13]: Should this be something like 'In the Traces tab, in your Checkly account settings'?

File: site/content/docs/traces-open-telemetry/importing-traces/_index.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+   
+2. Get the trace endpoint URL and the API key. Provide them to your preferred 3rd party OpenTelemetry backend.
+
+   ![Provide URL and header with authentication](/docs/images/integrations/otel/otel_export_traces_settings.png)
+
+
+
+3. Set up your OpenTelemetry collector to send only send Checkly relevant traces to Checkly. 
+
+   * [Setting up the OpenTelemetry Collector](/docs/importing-traces/sending-traces-otel-collector/)
```
Comment (drakirnosslin) [inline site/content/docs/traces-open-telemetry/importing-traces/_index.md:28]: This link gives me a 404 in the preview env.

File: site/content/docs/traces-open-telemetry/importing-traces/_index.md (hunk 2 above)
Comment (sujaya-sys) [inline site/content/docs/traces-open-telemetry/importing-traces/_index.md:13]:
```suggestion
In [the Traces tab in your Checkly account settings](https://app.checklyhq.com/settings/account/traces): 
```

Comment (MariadeAnton) [reply site/content/docs/traces-open-telemetry/importing-traces/_index.md:13]:
```suggestion
In [the Traces tab, in your Checkly account settings](https://app.checklyhq.com/settings/account/traces): 
```

Comment (MariadeAnton) [reply site/content/docs/traces-open-telemetry/importing-traces/_index.md:13]: Totally, great catch 💯

File: site/content/docs/traces-open-telemetry/importing-traces/sending-traces-otel-collector.md [hunk 4]
```diff
@@ -1,23 +1,28 @@
...
+  - "/docs/open-telemetry/sending-traces-otel-collector/"
 ---
 
 Using the OpenTelemetry collector is the best way to manage and route telemetry data to Checkly. It requires
 a little setting up, but it's worth it.
 <!--more-->
-In this guide we assume you already have an OTel collector running and configured. If you don't, please reference [the
-Open Telemetry Collector getting started documentation](https://opentelemetry.io/docs/collector/installation/)
+
+## Pre-requisites
```
Comment (drakirnosslin) [inline site/content/docs/traces-open-telemetry/importing-traces/sending-traces-otel-collector.md:21]: I think this is commonly spelled without a '-'

File: site/content/docs/traces-open-telemetry/importing-traces/_index.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+   ![Enable basic instrumentation](/docs/images/integrations/otel/otel_basic_instrumentation.png)
+
+   
+2. Get the trace endpoint URL and the API key. Provide them to your preferred 3rd party OpenTelemetry backend.
+
+   ![Provide URL and header with authentication](/docs/images/integrations/otel/otel_export_traces_settings.png)
+
+
+
+3. Set up your OpenTelemetry collector to send only send Checkly relevant traces to Checkly.
```
Comment (sujaya-sys) [inline site/content/docs/traces-open-telemetry/importing-traces/_index.md:26]:
```suggestion
3. Set up your OpenTelemetry collector to only send relevant traces to Checkly. 
```

File: site/content/docs/traces-open-telemetry/importing-traces/sending-traces-otel-collector.md (hunk 4 above)
Comment (MariadeAnton) [reply site/content/docs/traces-open-telemetry/importing-traces/sending-traces-otel-collector.md:21]: Thanks- I've been doing it with the dash for readibility but I can totally see that we've been saying Prerequisites in other pages like k8s. Changing it for consistency.
```suggestion
## Prerequisites
```

File: site/content/docs/traces-open-telemetry/importing-traces/_index.md (hunk 3 above)
Comment (MariadeAnton) [reply site/content/docs/traces-open-telemetry/importing-traces/_index.md:28]:
```suggestion
   * [Setting up the OpenTelemetry Collector](/docs/traces-open-telemetry/importing-traces/sending-traces-otel-collector/) 
```
this one should be the right onew :)

File: site/content/docs/traces-open-telemetry/importing-traces/_index.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+
+3. Set up your OpenTelemetry collector to send only send Checkly relevant traces to Checkly. 
+
+   * [Setting up the OpenTelemetry Collector](/docs/importing-traces/sending-traces-otel-collector/) 
+
+
+---
+
+After you've set this up, you should see traces coming in the [Traces page](https://app.checklyhq.com/traces). Moreover, when opening a check, you'll see a Traces section within it.
```
Comment (sujaya-sys) [inline site/content/docs/traces-open-telemetry/importing-traces/_index.md:33]:
```suggestion
After you've set this up, you should start seeing traces on the [Traces page](https://app.checklyhq.com/traces). Moreover, when opening a check, you'll see a Traces section within it.
```

File: site/content/docs/traces-open-telemetry/importing-traces/sending-traces-otel-collector.md [hunk 7]
```diff
@@ -1,23 +1,28 @@
...
+
+* An OpenTelemetry collector running and configured.
+
+If you need to set up OpenTelemetry for the first time, please use [the Open Telemetry Collector getting started documentation](https://opentelemetry.io/docs/collector/installation/)
```
Comment (sujaya-sys) [inline site/content/docs/traces-open-telemetry/importing-traces/sending-traces-otel-collector.md:25]:
```suggestion
If you need to set up OpenTelemetry for the first time, please use [the Open Telemetry Collector getting started documentation](https://opentelemetry.io/docs/collector/installation/).
```

File: site/content/docs/traces-open-telemetry/instrumenting-code/dot-net.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+menu:
+  platform:
+    parent: "Instrument your code with OpenTelemetry"
+beta: true
+aliases:
+  - "/docs/open-telemetry/instrumenting-code/dot-net"
+---
+.NET docs coming soon.
+
+* Check the canonical OpenTelemetry .NET documentation at [opentelemetry.io](https://opentelemetry.io/docs/languages/net/getting-started/)
```
Comment (sujaya-sys) [inline site/content/docs/traces-open-telemetry/instrumenting-code/dot-net.md:13]:
```suggestion
* Check the canonical OpenTelemetry .NET documentation at [opentelemetry.io](https://opentelemetry.io/docs/languages/net/getting-started/).
```

File: site/content/docs/traces-open-telemetry/_index.md [hunk 9]
```diff
@@ -1,47 +1,55 @@
 ---
-title: Getting started with OpenTelemetry
+title: Getting started with Checkly Traces and OpenTelemetry
```
Comment (sujaya-sys) [inline site/content/docs/traces-open-telemetry/_index.md:2]: Below we switched from "instrumenting" to "instrument" - could do the same here for consistency i.e. "Get started with.." - Not sure if this is a pattern we want to follow

File: site/content/docs/traces-open-telemetry/_index.md [hunk 10]
```diff
@@ -1,47 +1,55 @@
...
 correct OpenTelemetry SDKs and enable tracing inside Checkly. 
 [Book an onboarding session right here](https://calendly.com/maria-checkly/traces-onboarding)
 {{< /info >}}
 
-With OpenTelemetry enabled, you will have access to traces in all the places where it matters to more quickly resolve issues:
+With Traces configured, you will have access to traces in all the places where it matters to more quickly resolve issues:
 - **Check results:** resolve production outages faster by correlating failing checks with backend traces.
 - **Test sessions:** understand any failures during test session execution.
-- **Check Editors** get a live trace while building, editing and debugging check code.
+- **Check Editors:** get a live trace while building, editing and debugging check code.
```
Comment (sujaya-sys) [inline site/content/docs/traces-open-telemetry/_index.md:21]:
```suggestion
- **Check editors:** get a live trace while building, editing and debugging check code.
```

File: site/content/docs/traces-open-telemetry/how-it-works.md [hunk 11]
```diff
@@ -1,10 +1,12 @@
 ---
-title: How Checkly tracing works under the hood
+title: Understanding Checkly Traces
```
Comment (sujaya-sys) [inline site/content/docs/traces-open-telemetry/how-it-works.md:2]: Same as mentioned above, could stick to "Understand" here for consistency

File: site/content/docs/traces-open-telemetry/_index.md [hunk 12]
```diff
@@ -0,0 +1,63 @@
...
+	  title="I want to see traces in Checkly"
+	  description="Send your backend traces to Checkly to get contextualized check failure analysis."
+	  img="/docs/images/icons/opentelemetry.svg"
+	  link="/docs/traces-open-telemetry/importing-traces/"
+>}}
+{{< doc-card
+	  class="three-column-card"
+	  headerTag="h3"
+	  title="I want to export Check results to my OpenTelemetry setup"
+	  description="Export check results as traces for your 3rd party OTel tooling"
```
Comment (sujaya-sys) [inline site/content/docs/traces-open-telemetry/_index.md:50]:
```suggestion
	  description="Export check results as traces to your 3rd party OTel tooling"
```

File: site/content/docs/traces-open-telemetry/_index.md [hunk 13]
```diff
@@ -0,0 +1,63 @@
...
+>}}
+</div>
+
+<br>
+
+See this in action in the video below:
+
+<div style="position: relative; padding-bottom: 56.25%; height: 0;"><iframe src="https://www.loom.com/embed/30c143388ba54e9ba6b665dfbfe0d295?sid=8ad8d273-b0bb-48ca-b456-1b137384b9de" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe></div>
+
+If you want to learn more about how this all works, check out the [How it works under the hood](/docs/open-telemetry/how-it-works/) section.
```
Comment (sujaya-sys) [inline site/content/docs/traces-open-telemetry/_index.md:62]:
```suggestion
If you want to learn more about how this all works, check out the [Understand Checkly Traces](/docs/open-telemetry/how-it-works/) section.
```

-> Could adapt the naming here for consistency

**Review by sujaya-sys — COMMENTED** [review]
2 more minor issues, otherwise LGTM!

**Review by sujaya-sys — DISMISSED** [review]
LGTM!

**Review by sujaya-sys — APPROVED** [review]
🥳

**Pull request merged**